app/src/main/java/org/simple/clinic/scanid/ScanSimpleIdEffect.kt

```diff
@@ -17,3 +17,5 @@ data class ValidateShortCode(val shortCode: ShortCodeInput) : ScanSimpleIdEffect
 data class SendScannedIdentifierResult(val scannedId: ScanResult) : ScanSimpleIdEffect()
 
 data class SearchPatientByIdentifier(val identifier: Identifier) : ScanSimpleIdEffect()
+
+data class ParseJsonIntoPatientPrefillInfoObject(val text: String) : ScanSimpleIdEffect()
```

Comment:
Let's just call it `ParseScannedJson` or something like that. Instead of adding a specific class name at the end.